srcpkgs/rdma-core/template

```diff
@@ -1,20 +1,22 @@
 # Template file for 'rdma-core'
 pkgname=rdma-core
-version=23.3
+version=43.0
 revision=1
 build_style=cmake
-configure_args="-DENABLE_VALGRIND=OFF"
-hostmakedepends="pkg-config python3"
+configure_args="-DENABLE_VALGRIND=OFF
+ -DCMAKE_INSTALL_MODPROBEDIR=/usr/lib/modprobe.d"
+hostmakedepends="pkg-config python3 $(vopt_if docs 'pandoc python3-docutils')"
 makedepends="libnl3-devel eudev-libudev-devel python3-devel"
 short_desc="RDMA core userspace libraries and daemons"
 maintainer="Rich G <rich@richgannon.net>"
-license="GPL-2.0-or-later, BSD-2-Clause-FreeBSD"
+license="GPL-2.0-or-later, BSD-2-Clause"
 homepage="https://github.com/linux-rdma/rdma-core"
 distfiles="https://github.com/linux-rdma/rdma-core/releases/download/v${version}/rdma-core-${version}.tar.gz"
-checksum=0afb7aa2654ea914c3b5d12e6e8f2fc3f325b0a52da1febf4f66788bf0b2bb14
+checksum=05ae494efb2b6c86f9586ce75a9c959995cb4b122fcb36ed4a32f96c9650017d
+build_options="docs"
+desc_option_docs="Enable documentation"
```

Comment:
Enable this by default for platforms that provide `pandoc`.

Reply:
Done. Thank you!